content/en/blog/_posts/2025-XX-XX-endpoints-deprecation.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: blog
+title: "Endpoints formally deprecated in favor of EndpointSlices"
+slug: endpoints-deprecation
+date: 2025-XX-XX
+draft: true
+author: >
+  Dan Winship (Red Hat)
+---
+
+Since the addition of [EndpointSlices] several years ago, the
```

Comment:
I think it would be useful to give a bit more detail as to how long it has been since the addition.

```suggestion
Since the addition of [EndpointSlices] as alpha in v1.15 and later GA in v1.21, the
```

We could also potentially link to the original KEP https://github.com/kubernetes/enhancements/issues/752